src/colorizer/CanvasOverlay.ts

```diff
@@ -93,6 +99,36 @@ export default class CanvasOverlay implements IRenderCanvas {
     this.innerCanvas = canvas;
     this.canvasElement = document.createElement("canvas");
     this.canvasElement.style.display = "block";
+    // Let mouse events pass through to the inner canvas.
+    this.canvasElement.style.pointerEvents = "none";
+    // Ensure the canvas is rendered on top of the inner canvas.
+    this.canvasElement.style.position = "relative";
+    this.canvasElement.style.zIndex = "1";
+
+    // Set up DOM elements, which are structured like:
+    // canvasContainerDiv
+    //   canvasElement
+    //   innerCanvasContainerDiv
+    //     innerCanvas
+    this.canvasContainerDiv = document.createElement("div");
+    this.canvasContainerDiv.style.position = "relative";
+    this.canvasContainerDiv.style.width = "100%";
+    this.canvasContainerDiv.style.height = "100%";
+    this.canvasContainerDiv.style.overflow = "hidden";
+
+    this.innerCanvasContainerDiv = document.createElement("div");
+    this.innerCanvasContainerDiv.appendChild(this.innerCanvas.domElement);
+    this.innerCanvasContainerDiv.style.position = "absolute";
+    this.innerCanvasContainerDiv.style.top = "0px";
+    this.innerCanvasContainerDiv.style.left = "0px";
+    this.innerCanvasContainerDiv.style.zIndex = "0";
```

Comment:
optional thought: could you assign a single object in place of these individual property declarations?
```typescript
this.innerCanvasContainerDiv.style = {
    position: "absolute";
    top: "0px";
    // ...
};
```

Reply:
I can set it via the `cssText` property!